### PR TITLE
Add new GVC project ID, add Entity to display the comments and code formatted using lint formatter

### PR DESCRIPTION
**Pull request by jonaner**
The errors presented in the lint check are ignored in this current PR.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,26 +15,14 @@
...
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
 function addMessage() {
-  fetch('/data').then(response => response.json()).then((messages) => {
+  fetch('/data').then(response) => response.json()).then((messages) => {
```
Comment (ginellegaisano) [inline portfolio/src/main/webapp/script.js:19]: Missing open bracket around response.
((response) => response.json())

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,26 +15,14 @@
...
     const messageList = document.getElementById('past-messages');
     messageList.innerHTML = '';
     if (messages == null) {
       console.log('no information yet');
     } else {
-      messages.forEach(element => {
+      messages.forEach(element) => {
```
Comment (ginellegaisano) [inline portfolio/src/main/webapp/script.js:25]: Missing open bracket around element.
((element) => {
  messageList.appendChild(createListElement(element));
});

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>Jonathan's Portfolio</h1>
       <h2>Welcome to my Portfolio!</h2>
-      <input type="button" onclick="location.href='write-message.html'";"value="Write a comment"/>
-
+      <button onclick="location.href='write-message.html'">Write a Comment</button>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:13]: nit: `<button type="button ...`

File: portfolio/src/main/webapp/write-message.html [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
-      <p>Enter Username</p>
+      <h2>Enter Name</h2>
       <textarea name="username" placeholder="Type Username"></textarea>
 
-      <p>Enter Message Here</p>
+      <h2>Enter Message Here</h2>
       <textarea name="message-data" placeholder="Write a message about the webpage"></textarea>
-      <input type="submit"></input>
-      <br></br>
+      <input type="submit" />
```
[the remark this answers is not in the document]
Comment (jbir789) [reply portfolio/src/main/webapp/write-message.html:19]: I still see `<input type`, could you try `<button type="submit">`?

File: portfolio/src/main/webapp/write-message.html [hunk 5]
```diff
@@ -1,28 +1,29 @@
...
     <h1>Write A Message</h1>
-    <p>Prior Messages</p>
-
-    <ul id="past-messages"></ul>
-
+    <h2>Prior Messages</h2>
+    <h3>
+      <ul id="past-messages"></ul>
+    </h3>
     <form action="/data" method="POST">
```
[the remark this answers is not in the document]
Comment (jbir789) [reply portfolio/src/main/webapp/write-message.html:13]: I still see the capital cases here

**Comment by jbir789** [comment]
The button looks good, is there any concerns with it? I didn't build your branch, let me know if you want me to do it.

**Comment by jonaner** [comment]
There weren't any problems, I was just making sure that was the correct way to do that button. I did fix another error with the validate but the last error for the javascript probably can't be fixed.

**Comment by jbir789** [comment]
Yes, I am not sure why it's complaining the method is unused (probably because it isn't used in Javascript, so the lint check might not take a look at the HTML file), but you can ignore that one. Thanks for fixing all the other issues!

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -15,31 +15,23 @@
...
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
 function addMessage() {
```
Comment (jbir789) [inline portfolio/src/main/webapp/script.js:18]: It looks like you can add `// eslint-disable-line` after the method name to make the lint check ignore it

```
function addMessage() { // eslint-disable-line
```

**Pull request merged**